packages/sanity-studio/src/components/PathnameFieldComponent.tsx

```diff
@@ -186,9 +193,16 @@ export function PathnameFieldComponent(
   return (
     <Stack space={3}>
       <Stack space={2} flex={1}>
-        <Text size={1} weight="semibold">
-          {title}
-        </Text>
+        <Flex align="center" paddingY={1}>
+          <Text size={1} weight="semibold">
+            {title}
+          </Text>
+          {validation.length > 0 && (
+            <Box marginLeft={2}>
+              <FormFieldValidationStatus fontSize={1} placement="top" validation={validation} />
+            </Box>
+          )}
```

Comment:
Got this code from https://github.com/sanity-io/sanity/blob/251d592265efe31d8a4f2cd1655bc9bede03e66f/packages/sanity/src/core/form/components/formField/FormFieldHeaderText.tsx#L68-L72